python/tests/orderby_tests/orderby_int_varchar.py

```diff
@@ -0,0 +1,120 @@
+                       SELECT c1, c2
+                       FROM orderby_tbl_int_varchar
+                       ORDER BY c1 ASC, c2 ASC
+                       LIMIT 3"""
+        # Returns the following:
+        # [{'c1': None, 'c2': 'fred says konichiwa'}, {'c1': None, 'c2': 'see you later'}, {'c1': 1, 'c2': 'ferris says ciao'}]
+
+
+class orderby_asc_int_varchar(TstView):
+    def __init__(self):
+        # Validated on SQLite
+        self.data = []
+        self.sql = """CREATE MATERIALIZED VIEW orderby_asc_int_varchar AS
+                       SELECT c1, c2
+                       FROM orderby_tbl_int_varchar
+                       ORDER BY c1 ASC
+                       LIMIT 3"""
+        # Returns the following:
+        # [{'c1': None, 'c2': 'see you later'}, {'c1': None, 'c2': 'fred says konichiwa'}, {'c1': 1, 'c2': 'ferris says ciao'}]
+
+
+class orderby_asc_varchar_asc_int(TstView):
+    def __init__(self):
+        # Validated on SQLite
+        self.data = []
+        self.sql = """CREATE MATERIALIZED VIEW orderby_asc_varchar_asc_int AS
+                       SELECT c2, c1
+                       FROM orderby_tbl_int_varchar
+                       ORDER BY c2 ASC, c1 ASC
+                       LIMIT 7"""
+        # Returns the following:
+        # [{'c2': None, 'c1': 2}, {'c2': None, 'c1': 3}, {'c2': 'bye bye, friend!!', 'c1': 2}, {'c2': 'ferris says ciao', 'c1': 1}, {'c2': 'fred says konichiwa', 'c1': None}, {'c2': 'hello', 'c1': 1}, {'c2': 'hello', 'c1': 3}]
+
+
+class orderby_asc_varchar_int(TstView):
+    def __init__(self):
+        # Validated on SQLite
+        self.data = []
+        self.sql = """CREATE MATERIALIZED VIEW orderby_asc_varchar_int AS
+                       SELECT c2, c1
+                       FROM orderby_tbl_int_varchar
+                       ORDER BY c2 ASC
+                       LIMIT 7"""
+        # Returns the following:
+        # [{'c2': None, 'c1': 2}, {'c2': None, 'c1': 3}, {'c2': 'bye bye, friend!!', 'c1': 2}, {'c2': 'ferris says ciao', 'c1': 1}, {'c2': 'fred says konichiwa', 'c1': None}, {'c2': 'hello', 'c1': 3}, {'c2': 'hello', 'c1': 1}]
+
+
+class orderby_asc_no_nulls_int_and_var(TstView):
+    def __init__(self):
+        # Validated on SQLite
+        self.data = []
+        self.sql = """CREATE MATERIALIZED VIEW orderby_asc_no_nulls_int_and_var AS
+                       SELECT c1, c2
+                       FROM orderby_tbl_int_varchar
+                       WHERE c1 IS NOT NULL AND c2 IS NOT NULL
+                       ORDER BY c1 ASC, c2 ASC
+                       LIMIT 3"""
+
+
+class orderby_desc_int_desc_varchar(TstView):
+    def __init__(self):
+        # Validated on SQLite
+        self.data = []
+        self.sql = """CREATE MATERIALIZED VIEW orderby_desc_int_desc_varchar AS
+                       SELECT c1, c2
+                       FROM orderby_tbl_int_varchar
+                       ORDER BY c1 DESC, c2 DESC
```

Comment:
For each column we try:
- ASC
- DESC
Independently:
- nothing
- NULLS FIRST
- NULLS LAST
This means 2 * 3 = 6 versions for each column, so there should be 36 tests for 2 columns in this file